src/main/webapp/app/orion/management/orion-course-management-exercises.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'jhi-orion-course-management-exercises',
+    templateUrl: './orion-course-management-exercises.component.html',
```

Comment:
[nit] What do you think, is it better inline html here as the component is really small and has no typescript code? Like here: https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/shared/alert/alert.component.ts#L6-L17

Reply:
Makes sense, inlined the html :+1: